NDTensors/src/dense/tensoralgebra/contract.jl

```diff
@@ -379,7 +392,7 @@ function _contract!(
 
   if props.permuteC
     pC = NTuple{NC,Int}(props.PC)
-    Cr = ReshapedArray(CM.parent, props.newCrange, ())
+    Cr = reshape(CM.parent, props.newCrange)
```

Comment:
Looks like `CM.parent` is causing issues since `CM` isn't a wrapped array anymore. Probably can just change to `CM`.

Reply:
@kmp5VT I tried changing it, let's see if it passes.